pages/my-rides/index.js

```diff
@@ -16,73 +16,151 @@ import {
+        <StyledLinkWrapper>
+          <StyledLink href={`/`}>Zurück zu allen Fahrten</StyledLink>
+        </StyledLinkWrapper>
+      </>
+    );
   }
 
   return (
     <main>
       <Heading>landwaerts</Heading>
       <SecondHeading>Meine Fahrten</SecondHeading>
-      <SecondHeading $sticky>Anstehend:</SecondHeading>
-
-      <ul role="list">
-        {bookedRides.map((ride) => (
-          <li key={ride.id}>
-            <StyledCard>
-              <StyledRideInformation $font>
-                {ride.date}, {ride.time}
-              </StyledRideInformation>
-              <StyledDriverInformation>
-                <Image
-                  src={"/images/charlesdeluvio-K4mSJ7kc0As-unsplash.jpg"}
-                  alt="profile pic"
-                  width={100}
-                  height={100}
-                />
-                <StyledSpanForDetailPage $marginbottom>
-                  {ride.driver}
-                </StyledSpanForDetailPage>
-                <StyledSpanForDetailPage>{ride.car}</StyledSpanForDetailPage>
-              </StyledDriverInformation>
-              <dl>
-                <StyledDescriptionTerm>Kontakt:</StyledDescriptionTerm>
-                <StyledDescriptionDetails>
-                  {ride.contact}
-                </StyledDescriptionDetails>
-                <StyledDescriptionTerm>Von:</StyledDescriptionTerm>
-                <StyledDescriptionDetails>
-                  {ride.departure}
-                </StyledDescriptionDetails>
-                <StyledDescriptionTerm>Treffpunkt:</StyledDescriptionTerm>
-                <StyledDescriptionDetails>
-                  {ride.meetingpoint}
-                </StyledDescriptionDetails>
-                <StyledDescriptionTerm>Nach:</StyledDescriptionTerm>
-                <StyledDescriptionDetails>
-                  {ride.destination}
-                </StyledDescriptionDetails>
-                <StyledDescriptionTerm>Sitzplätze:</StyledDescriptionTerm>
-                <StyledDescriptionDetails>
-                  {ride.seats}
-                </StyledDescriptionDetails>
-                <StyledDescriptionTerm>Preis:</StyledDescriptionTerm>
-                <StyledDescriptionDetails>
-                  {ride.price}
-                  {ride.currency}
-                </StyledDescriptionDetails>
-                <StyledPaymentInformation $marginbottom>
-                  Achtung!
-                </StyledPaymentInformation>
-                <StyledPaymentInformation>
-                  Bitte passend und in bar zahlen!
-                </StyledPaymentInformation>
-              </dl>
-            </StyledCard>
-          </li>
-        ))}
-      </ul>
+      {filteredRides.length > 0 && (
+        <>
+          <SecondHeading $sticky>Hier bin ich Fahrer*in:</SecondHeading>
+          <ul role="list">
+            {filteredRides.map((ride) => (
+              <li key={ride.id}>
+                <StyledCard>
+                  <StyledRideInformation $font>
```

Comment:
I have never before seen `$font` etc. as a boolean prop for styling - it works and you can keep it, I guess. I just wanted to point that out.